README.md

```diff
@@ -7,11 +7,23 @@ Kubernetes Custom Resource and Controller for generating and signing SAML metada
 ## Requirements
 
 - docker
-- [kubebuilder](https://book.kubebuilder.io/getting_started/installation_and_setup.html)
+- [kubebuilder](https://book.kubebuilder.io/quick-start.html#installation)
 - [kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md)
+- [gds-cli](https://github.com/alphagov/gds-cli)
 
 ## Development
 
+Use `gds-cli` to update `kubeconfig` to refer to the target cluster:    
+
+`gds-cli <cluster> update-kubeconfig`
+
+This will generate a `kubeconfig` in `~/.gds/<cluster>-<cluster>.kubeconfig`
+
+Export this config:
+`exportKUBECONFIG=~/.gds/<cluster>-<cluster>.kubeconfig`
```

Comment:
`export KUBECONFIG ...

Reply:
I'm gonna leave this for now so I don't dismiss Lewis's review as he's on holiday now. I'll fix it up in another PR.